yarn.lock

```diff
@@ -5815,7 +5839,7 @@ cheerio-select@^2.1.0:
     domhandler "^5.0.3"
     domutils "^3.0.1"
 
-cheerio@1.0.0-rc.12, cheerio@^1.0.0-rc.12:
+cheerio@1.0.0-rc.12:
```

Comment:
And see, cheerio is still there, so my previous comment makes even more sense now. 

nit: Can you say why is still there? `yarn why cheerio`?

Reply:
Cheerio is still used by cozy-konnector-libs and since this is a monorepo, we still see it in yarn.lock. But it won't be included in cozy-ccc-libs in the end